subsys/ieee802154/ieee802154_nrf5_cfg.c

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <init.h>
+#include <irq.h>
+#include <kernel.h>
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
```

Comment:
I don't see `LOG_MODULE_NAME` defined, you can simply assign some name in this macro. Plus, you specify the log level in Kconfig,  you should use it then, something like:
`LOG_MODULE_REGISTER(ieee802154_nrf5_cfg, CONFIG_RD_CFG_LOG_LEVEL)`

Reply:
fixed